pkg/operator/starter.go

```diff
@@ -267,7 +267,6 @@ var RevisionConfigMaps = []revision.RevisionResource{
 	{Name: "etcd-peer-client-ca"},
 	{Name: "etcd-metrics-proxy-serving-ca"},
 	{Name: "etcd-metrics-proxy-client-ca"},
-	{Name: "restore-etcd-pod"},
```

Comment:
What's the difference between these 2 sets?

Reply:
With the change restore-etcd-pod gets generated under the revision number directory, instead of in etcd-certs (certConfigMaps).